lessons/Algebra2/periodic1-exploring-rides/langs/en-us/index.adoc

```diff
@@ -135,142 +140,169 @@ Students begin to reason about points on the circumference of a circle, with the
+@A{We could use the Pythagorean Theorem, with @math{x = y}: @hspace{1em} @math{x^2 + x^2 = 1^2}}
 }
 
 @lesson-instruction{
 - With a partner, finish @printable-exercise{unit-clock.adoc}.
-- The end of the page will direct you to use the the link I shared to the Desmos File *Exploring Periodic Functions*.}
+- *When you have completed the graph at the bottom*, open @starter-file{alg2-unit-clock}
+}
 
 @slidebreak
 
-@right{@image{images/pizza-slice2.png, 100}}As the point (A, B) travels around the circumference of a circle, it reflects a changing angle @math{θ}. It can be helpful to think of this as a pizza slice, with @math{θ} as the angle at the tip of the slice, and the edge of the crust as the amount of the circumference the point has traveled.
+@right{@image{images/pizza-slice2.png, 100}}As the point (A, B) travels around the circumference of a circle, it reflects a changing angle @math{θ}. Think of a pizza slice, with @math{θ} as the angle at the tip of the slice, and the crust as the amount of the circumference the point has traveled.
 
 @QandA{
-In our clock example, we divide the circle into twelve "slices", each representing one hour. How many slices would represent 15 minutes? An hour and a half? slice?
+In our clock example, we divide the circle into twelve "slices", each representing one hour. How many of those slices would represent 15 minutes? An hour and a half?
 
 Of course, there are other ways besides 12 slices of "hours" to measure this angle!
 @Q{Can you think of another measure that divides a circle up differently?}
-@A{_Minutes_, which divide our circle up into 720 slices instead of 12. We could imagine one-and-a-half of these slices representing 90 seconds, or 2 slices for 120 seconds.
+@A{_Minutes_, which divide our 12-hours into 720 slices. We could imagine one-and-a-half of these slices representing 90 seconds, or 2 slices for 120 seconds.
 ** _Degrees_, divide a circle up into 360 slices instead of 12. How many minutes are represented by 1 degree? 2.5 degrees?
 ** _Compass Directions_ like North, South, East, and West, which divide our circle up into 4 slices instead of 12. How many slices represent the angle between North and South? West and Southwest?
 }
-@Q{On our graph, would the shape of the curve change if we labeled the x-axis with 360 degrees instead of 12 hours?}
-@A{No -- all the intervals remain constant, so the only change is the *labels* on the x-axis.}
 }
 
 @slidebreak
 
-==== Radians
+Would the shape of the curves change if we used 360 degrees instead of 12 hours?
+
+@lesson-instruction{
+In our clock animation we have 12 "slices", and drawing 4 evenly-spaced labels around the clock.
+
+- Return to the @starter-file{alg2-unit-clock}, change `num-slices` to 360, and click "Run". Did the curves change?
+- Take a minute to play with `num-slices` and `num-labels`, making sure that `num-labels` divides evenly into `num-slices` with _no remainder!_
+- Can you divide the clock into 70 slices? 92?
```

Comment:
@schanzer I'm unclear what you want people to say here. I can type num-slices 70 and num-labels 5 into the starter file. But is this evenly divided? the clock, yes. the dots on the graph, less clear. Do we want to ask a more concrete question the elicits 5 as a possible response? Definitely need to provide more guidance for teachers!
<img width="1074" alt="Screenshot 2025-03-07 at 4 25 45 PM" src="https://github.com/user-attachments/assets/14e4cb4f-3f58-466f-9cb2-f2b0843bc08d" />

Reply:
Someone decided that there are 360 degrees in a circle.
Someone decided that there are 2pi radians in a circle. 
What number of slices would you use for a circle if you were in charge of making a new labeling system? what would the num-labels be - sketch the clock as it would appear in this pyret file.